### PR TITLE
[14.0][IMP] account_financial_report: VAT report fields translation

### PR DESCRIPTION
**Pull request by toita86**
Currently in the VAT report, the date does not correspond to the localization (it should appear as 'DD-MM-AAA' if the localization is set in Italy for ex.). Additionally,  the column `based_on` is not appearing with the translated label.
With this improvement things work based on the localization.

### CODE REVIEW

File: account_financial_report/report/templates/vat_report.xml [hunk 1]
```diff
@@ -76,7 +76,7 @@
                                 t-raw="tag_or_group['net']"
                                 t-options="{'widget': 'monetary', 'display_currency': res_company.currency_id}"
                             />
-                        </div>
+                            </div>
```
Comment (HekkiMelody) [inline account_financial_report/report/templates/vat_report.xml]: **chore**: I think previous formatting was correct. In general, to keep the PR clean, we tend to exclude modifications unrelated to the PR at hand. Formatting fixes can be done in their own commit, if necessary, but in this case they should be excluded.

**Review by HekkiMelody — CHANGES_REQUESTED** [review]
Code review: LGTM, just clean up the changes please.

File: account_financial_report/report/templates/vat_report.xml (hunk 1 above)
Comment (toita86) [reply account_financial_report/report/templates/vat_report.xml]: Fixed right now, thank you for pointing this out!

**Review by HekkiMelody — APPROVED** [review]
Code review, LGTM

**Review by francesco-ooops — APPROVED** [review]
Functional ok!

@simahawk what do you think?

**Comment by francesco-ooops** [comment]
@OCA/accounting-maintainers anyone available for review/merge? Thanks!

**Comment by francesco-ooops** [comment]
@pedrobaeza could you check? thanks!

**Review by pedrobaeza — COMMENTED** [review]
All the dates thing is unrelated to the PR title. It should be explained and isolated in a commit.

**Comment by francesco-ooops** [comment]
@pedrobaeza should be fixed now

**Comment by pedrobaeza** [comment]
The commit message about the dates is still no explaining why these changes.

**Comment by toita86** [comment]
@pedrobaeza I have updated the commit message, let me know if its ok.

**Comment by pedrobaeza** [comment]
I still don't see any explanation in the commit message:

imagen

Why do you do `fields.Date.from_string`? Steps to reproduce any problem or issue?

**Comment by toita86** [comment]
My bad, I have added the commit description explaining what was the problem and the fix for it.

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge minor

**Pull request merged**